src/Features/Core/Portable/InheritanceMargin/AbstractInheritanceMarginService_Helpers.cs

```diff
@@ -571,39 +546,10 @@ internal abstract partial class AbstractInheritanceMarginService
                 projectState?.NameAndFlavor.name ?? projectState?.Name);
         }
 
-        private static ImmutableArray<ISymbol> GetImplementedSymbolsForTypeMember(
-            ISymbol memberSymbol,
-            ImmutableArray<ISymbol> overriddenSymbols)
```

Comment:
what happened with this guy? :)

Reply:
I think it has been replaced by ImplementedSymbolAndOverriddenSymbolsFinder